backend/btrixcloud/db.py

```diff
@@ -216,7 +215,6 @@ async def create_indexes(
     await coll_ops.init_index()
     await invite_ops.init_index()
     await user_manager.init_index()
-    await page_ops.init_index()
```

Comment:
Won't this mean that new deployments won't have page indices created? I think we might want to check if it already exists rather than just removing the `init_index()` entirely

Reply:
We also haven't yet removed the code that will drop the index, which at this point would force us to recreate it

Reply:
It turns out `create_index` is idempotent, so I've reversed the logic in this PR. Rather than removing the call to create page indices, we now remove the logic that drops the page indices after successful migration.